README.md

```diff
@@ -222,43 +222,39 @@ to be compliant with it, to reduce developer learning curve, make our preexistin
 A very basic [benchmark](./benchmark/index.js) with the most common use case has 
 been setup to compare with [`bunyan`](https://github.com/trentm/node-bunyan)
 
-Running on a MAC OS X Yosemite, 2,5 GHz Intel Core i5, 8 GB 1333 MHz DDR3, SSD disk, node 4.2.2
+Running on a MAC OS X Yosemite, 2,5 GHz Intel Core i5, 8 GB 1333 MHz DDR3, SSD disk, node 6.10.0
  
 ```
 $ cd benchmark; npm start
-               
-> benchmarklogops@1.0.0 start /Users/javier/Documents/Proyectos/logops/benchmark
-> npm run tee && npm run file && npm run null && rm out.log
-
-
+ 
 > benchmarklogops@1.0.0 tee /Users/javier/Documents/Proyectos/logops/benchmark
 > node index.js | tee -a out.log > /dev/null
 
-logops x 39,560 ops/sec ±3.00% (75 runs sampled)
-bunyan x 27,365 ops/sec ±2.23% (79 runs sampled)
-Basic logging: Fastest is logops
-logops x 73,150,310 ops/sec ±1.64% (79 runs sampled)
-bunyan x 1,569,549 ops/sec ±3.67% (78 runs sampled)
+logops x 70,675 ops/sec ±11.89% (65 runs sampled)
+bunyan x 81,981 ops/sec ±4.76% (70 runs sampled)
+Basic logging: Fastest is bunyan,logops
```

Comment:
It's the output from the benchmark tool. I guess it's related with the deviation in the samples, so logops -11.89% is aprox bunyan +4.76%